src/templates/ja/2019/index.html

```diff
@@ -1,32 +1,32 @@
 {% extends "base/2019/index.html" %}
 
-{% block title %}The {{ year }} Web Almanac{% endblock %}
-{% block description %}The Web Almanac is an annual state of the web report combining the expertise of the web community with the data and trends of the HTTP Archive.{% endblock %}
+{% block title %}The {{ year }} Web年鑑{% endblock %}
+{% block description %}Web年鑑は、Webコミュニティの専門知識とHTTPアーカイブのデータやトレンドを組み合わせた、毎年恒例のWeb状態のレポートです。{% endblock %}
 
-{% block twitter_image_alt %}The {{ year }} Web Almanac{% endblock %}
+{% block twitter_image_alt %}The {{ year }} Web年鑑{% endblock %}
```

Comment:
Don't translate Web Almanac. But what about the word "The" at the beginning?

```suggestion
{% block twitter_image_alt %}The {{ year }} Web Almanac{% endblock %}
```

Reply:
I fixed.